setup.py

```diff
@@ -72,7 +72,7 @@ def build_parser():
         help="""Caches the arguements sent to setup.py
         and reloads them on the next invocation.  This argument is not cached""",
     )
-
+    # --cmake-args="-DWITH_BULLET=ON"
```

Comment:
This is needed only if users want to build with bullet. Not sure if we should always set it to `ON` though